ClosedXML/Excel/Cells/XLCell.cs

```diff
@@ -244,8 +244,6 @@ internal IXLCell SetValue<T>(T value, bool setTableHeader)
 
             SetInternalCellValueString(parsedValue, validate: true, parseToCachedValue: false);
 
-            CachedValue = null;
```

Comment:
I'm skeptical about this line's removal.

Reply:
https://github.com/ClosedXML/ClosedXML/pull/1389/commits/9a468dcb1068ba3f387da1b964ef84e668b0d6f1#diff-3b8a562c15b0aca901df66d96df87e10R223 maintains this.